app/Services/OrderService.php

```
@@ -81,6 +82,23 @@ public function completeOrder(Order $order, array $paymentData = [])
         $order->save();
         // increase sales for each order items
         foreach ($order->items()->with('product')->get() as $item) {
+            // generate license code for order item
+            $licenseCode = Str::uuid();
```

PHPMD (2.15.0)

Avoid using static access to class '\Str' in method 'completeOrder'. (Clean Code Rules)

(StaticAccess)